sentry/src/test/java/io/sentry/HubTest.kt

```diff
@@ -1213,19 +1213,24 @@ class HubTest {
     @Test
     fun `when startTransaction and profiling is enabled, transaction is profiled only if sampled`() {
         val mockTransactionProfiler = mock<ITransactionProfiler>()
+        val unsampledHub = generateHub {
```

Comment:
Why did we rename this?

Reply:
i wanted to let the hub make decisions based on the options, but it would make more sense to simply change the `TracesSamplingDecision()` call